pyInterface/testMC.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
+
+### BEGIN CONFIGURATION ###
+
+	### BEGIN PHASE SPACE ###
+
+		# mass bin (MeV)
+		MASS=1800
+		BINWIDTH=20
+
+		# number of phase space events to generate 
+		NMB_PS_EVENTS=500	
+
+	#-- END PHASE SPACE --#
+
+	### BEGIN PARTICLE DATA ###
+
+		# particle data table
+		PARTICLE_DATA_TABLE="${ROOTPWA}/particleData/particleDataTable.txt"	
+
+	#-- END PARTICLE DATA --#
+
+	### BEGIN PSEUDO DATA ###
+
+		# number of weighted pseudo data events to generate
+		NMB_PSEUDO_EVENTS=500
+
+	#-- END PSEUDO DATA --#
+
+
+#-- END CONFIGURATION --#
+
+
+### BEGIN PREPARATION ###
+
+if [ -d "${ROOTPWA}" ]; then
+	echo "RootPwa installation found in: ${ROOTPWA}"
+	echo "Starting MC test ..."
+else
+	echo "Error no installation of RootPwa found! Aborting."
+	exit 1
+fi
+
+TESTDIR=${PWD}/ROOTPWA_MC_TEST
+
+if [ -d "${TESTDIR}" ]; then
+	echo "Test directory ${TESTDIR} already exists! Aborting."
+	exit 1
+fi
+
+echo "Creating directories ..."
+
+mkdir "${TESTDIR}"
+mkdir "${TESTDIR}/data"
+mkdir "${TESTDIR}/weighted_mc_data"
+mkdir "${TESTDIR}/keyfiles"
+mkdir "${TESTDIR}/reference_fit"
+mkdir "${TESTDIR}/fits"
+mkdir "${TESTDIR}/log"
+
+echo "Test directory and subfolders created."
+echo "Changing to test directory."
+
+cd "${TESTDIR}"
+
+exec > >(tee -i ./log/logfile.txt)
+exec 2>&1
+
+echo "Copying necessary files to test directory ..."
+
+cp "${ROOTPWA}/userAnalysisWorkspace/3pi.--+/generator_noBeamSimulation.conf" "./"
+cp "${ROOTPWA}/userAnalysisWorkspace/3pi.--+/keyfiles/template.key" "./"
```

Comment:
with commit dff45527e01ccfc78fb3e8bb0d9d5c96fd17262e copying `template.key` would no longer be required if an `export TEMPLATE_KEY_FILES="${ROOTPWA}/userAnalysisWorkspace/3pi.--+/keyfiles/template.key"` is added before line 94.